AssetManagementTestProject/TestSetup/NUnitWebTestSetup.cs

```diff
@@ -25,12 +35,40 @@ public void SetUp()
         Asserter = new Asserter.Asserter();
         MenuBarLeft = new LeftMenuPage();
         ChangePw1stTime = new ChangePassword1stTimePage();
+        ChangePassword = new ChangePasswordPage();
+
+        AuthorizationService = new AssetManagementAPIServices();
+        Token = AuthorizationService.ReturnLoginToken(Constant.ADMIN_USERNAME_HN, Constant.ADMIN_PASSWORD_HN);
+        APIService = new AssetManagementAPIServices();
+        NewUser = APIService.ReturnNewUser(Constant.NEW_STAFF_HN, Token);
+        FirstTimeLoginData newLoginData = new FirstTimeLoginData();
+        newLoginData.NewUser = NewUser;
+        string newAdminUsername = newLoginData.GetUsername();
+        string newAdminPassword = newLoginData.GetPassword();
+        LoginPage?.Login(newAdminUsername, newAdminPassword);
+        ChangePw1stTime = new ChangePassword1stTimePage();
```

Comment:
Pre-Condition does not mean it should be in test setup, I think you can include login steps into your test cases, and then add comments to let people know where your pre-condition is in your test cases